rules/windows/privilege_escalation_via_rogue_named_pipe.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2021/10/13"
+maturity = "production"
+updated_date = "2021/10/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a privilege escalation attempt via rogue named pipe impersonation. An adversary may abuse this technique by
+masquerading as a known named pipe and manipulating a privileged process to connect to it.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Privilege Escalation via Rogue Named Pipe Impersonation"
+note = """Named Pipe Creation Events need to be enabled in you Sysmon configuration by including the following settings
+condition equal "contains" and keyword equal "pipe".
+"""
+references = [
+    "https://itm4n.github.io/printspoofer-abusing-impersonate-privileges/",
+    "https://github.com/zcgonvh/EfsPotato",
+    "https://twitter.com/SBousseaden/status/1429530155291193354"
+]
+risk_score = 73
+rule_id = "76ddb638-abf7-42d5-be22-4a70b0bf7241"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.action : "Pipe Created*" and
```

Comment:
What is the `event.category`? Use that would be preferred if possible

Reply:
![image](https://user-images.githubusercontent.com/64742097/142156152-86010d68-84a1-48e2-8734-736b4eb45d1c.png)